electricpy/__init__.py

```diff
@@ -2294,98 +1552,10 @@ def thermocouple(V, coupletype="K", fahrenheit=False, cjt=None, To=None,
-            "R": [[1.3054315E+02, 5.4188181E+02, 1.0382132E+03, 1.5676133E+03],
-                  [8.8333090E-01, 4.9312886E+00, 1.1014763E+01, 1.8397910E+01],
-                  [1.2557377E+02, 9.0208190E+01, 7.4669343E+01, 7.1646299E+01],
-                  [1.3900275E+02, 6.1762254E+00, 3.4090711E+00, -1.0866763E+00],
-                  [3.3035469E+01, -1.2279323E+00, -1.4511205E-01, -2.0968371E+00],
-                  [-8.5195924E-01, 1.4873153E-02, 6.3077387E-03, -7.6741168E-01],
-                  [1.2232896E+00, 8.7670455E-02, 5.6880253E-02, -1.9712341E-02],
-                  [3.5603023E-01, -1.2906694E-02, -2.0512736E-03, -2.9903595E-02],
-                  [0.0, 0.0, 0.0, -1.0766878E-02]],
-            "S": [[1.3792630E+02, 4.7673468E+02, 9.7946589E+02, 1.6010461E+03],
-                  [9.3395024E-01, 4.0037367E+00, 9.3508283E+00, 1.6789315E+01],
-                  [1.2761836E+02, 1.0174512E+02, 8.7126730E+01, 8.4315871E+01],
-                  [1.1089050E+02, -8.9306371E+00, -2.3139202E+00, -1.0185043E+01],
-                  [1.9898457E+01, -4.2942435E+00, -3.2682118E-02, -4.6283954E+00],
-                  [9.6152996E-02, 2.0453847E-01, 4.6090022E-03, -1.0158749E+00],
-                  [9.6545918E-01, -7.1227776E-02, -1.4299790E-02, -1.2877783E-01],
-                  [2.0813850E-01, -4.4618306E-02, -1.2289882E-03, -5.5802216E-02],
-                  [0.0, 1.6822887E-03, 0.0, -1.2146518E-02]],
-            "T": [[-1.9243000E+02, -6.0000000E+01, 1.3500000E+02, 3.0000000E+02],
-                  [-5.4798963E+00, -2.1528350E+00, 5.9588600E+00, 1.4861780E+01],
-                  [5.9572141E+01, 3.0449332E+01, 2.0325591E+01, 1.7214707E+01],
-                  [1.9675733E+00, -1.2946560E+00, 3.3013079E+00, -9.3862713E-01],
-                  [-7.8176011E+01, -3.0500735E+00, 1.2638462E-01, -7.3509066E-02],
-                  [-1.0963280E+01, -1.9226856E-01, -8.2883695E-04, 2.9576140E-04],
-                  [2.7498092E-01, 6.9877863E-03, 1.7595577E-01, -4.8095795E-02],
-                  [-1.3768944E+00, -1.0596207E-01, 7.9740521E-03, -4.7352054E-03],
-                  [-4.5209805E-01, -1.0774995E-02, 0.0, 0.0]]}
     # Load Data Into Terms
-    if To == None:
-        To = data[coupletype][0][select]
-    if Vo == None:
-        Vo = data[coupletype][1][select]
-    if P1 == None:
-        P1 = data[coupletype][2][select]
-    if P2 == None:
-        P2 = data[coupletype][3][select]
-    if P3 == None:
-        P3 = data[coupletype][4][select]
-    if P4 == None:
-        P4 = data[coupletype][5][select]
-    if Q1 == None:
-        Q1 = data[coupletype][6][select]
-    if Q2 == None:
-        Q2 = data[coupletype][7][select]
-    if Q3 == None:
-        Q3 = data[coupletype][8][select]
+    var_list = ['To', 'Vo', 'P1', 'P2', 'P3', 'P4', 'Q1', 'Q2', 'Q3']
+    for i in range(9):
+        exec(f'{var_list[i]} = {var_list[i]} or {THERMO_COUPLE_DATA[coupletype][i][select]}')
```

Comment:
Let's see if we can avoid the use of the `exec` function if at all possible. Though it's not terribly relevant in a scientific/data-science package, it still can pose a security risk and vulnerability, no matter how minute we may view that.

**References:**

* https://blog.finxter.com/python-exec/
* https://stackoverflow.com/questions/9672791/how-to-safely-use-exec-in-python

- [ ] Refactor to avoid use of `exec(...)`